go.mod

```diff
@@ -3,108 +3,62 @@ module github.com/kairos-io/kairos/provider-kubeadm
 go 1.18
 
 require (
-	github.com/kairos-io/kairos v1.24.3-56.0.20230210083748-e738e751aa3a
+	github.com/kairos-io/kairos-sdk v0.0.2-0.20230414094028-0c9d2bd9e6ae
```

Comment:
The kairos-sdk repo [doesn't have a v0.0.2 tag](https://github.com/kairos-io/kairos-sdk/tags) afaict. I wonder how `go mod` decides to refer to it like this :thinking:

Reply:
very good question :man_shrugging: